pkg/controller/storagecluster/storagecluster.go

```diff
@@ -1450,6 +1451,10 @@ func (c *Controller) CreatePodTemplate(
 		}
 	}
 
+	if pxutil.IsOpenshift(cluster) {
+		newTemplate.Annotations[component.OpenshiftRequiredSCCAnnotation] = component.PxSCCName
```

Comment:
Can you write a unit test to validate that the pod created when is-openshift set is true has the required-scc annotation?

Reply:
Will add in seperate PR as discussed